File: src/com/infoecos/cn/destiny/lib/时/节气.java
package com.infoecos.cn.destiny.lib.时;

import java.util.Date;

public class 节气 {
	// 节气的时间。
	private Date solarTermDate;
	// 节气名。
	private String name;

	public Date getSolarTermDate() {
		return solarTermDate;
	}

	public void setSolarTermDate(Date solarTermDate) {
		this.solarTermDate = solarTermDate;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

}

File: libs/destiny/时/节气.php
<?php
namespace DchLib;

class 节气 {
	// 节气的时间。
	private $m_solarTermDate;
	// 节气名。
	private $m_name;

	public function getSolarTermDate() {
		return $this->m_solarTermDate;
	}

	public function setSolarTermDate($solarTermDate) {
		$this->m_solarTermDate = $solarTermDate;
	}

	public function getName() {
		return $this->m_name;
	}

	public function setName($name) {
		$this->m_name = $name;
	}

}
